### PR TITLE
Training overview add images and content

### PR DESCRIPTION
**Pull request by mattgraygithub**
Work in progress for adding content and imagery for the [training overview page](https://codurance-online.leankit.com/card/1061802465)

### CODE REVIEW

File: src/assets/custom/css/custom-global-styles.scss [hunk 1]
```diff
@@ -20,6 +20,7 @@ body {
   /* 0 0 0 1 */
   font-size: 15px;
   font-family: proxima-nova, sans-serif;
+  overflow-x: hidden;
```
Comment (mattgraygithub) [inline src/assets/custom/css/custom-global-styles.scss]:
```suggestion
```

Comment (mattgraygithub) [reply src/assets/custom/css/custom-global-styles.scss]: no longer needed

**Review by AurelijaZuba — APPROVED** [review]

**Review by cherryt — APPROVED** [review]
Nice work :D

**Review by Dan-Bird — APPROVED** [review]
Seems good 👍

**Pull request merged**